### PR TITLE
Pce sc comparison

### PR DESCRIPTION
**Pull request by DavidPCoster**
I think we are ready to merge into the dev branch

### CODE REVIEW

File: easyvvuq/analysis/sc_analysis.py [hunk 1]
```diff
@@ -679,22 +652,76 @@ def get_moments(self, qoi):
         logging.debug('done')
         return mean_f, var_f
 
+    # def sc_expansion(self, samples, x):
```
Comment (orbitfold) [inline easyvvuq/analysis/sc_analysis.py]: If it's commented out it should be removed. It is going to be preserved in the git history anyway.

**Review by orbitfold — CHANGES_REQUESTED** [review]
See the comment I left on sc_analysis.py

File: easyvvuq/analysis/sc_analysis.py (hunk 1 above)
Comment (DavidPCoster) [reply easyvvuq/analysis/sc_analysis.py]: This was done by @wedeling and is not something I changed ... I would prefer not to remove the lines as I assume he had a good reason for keeping them ...

**Review by wedeling — APPROVED** [review]

File: easyvvuq/analysis/sc_analysis.py (hunk 1 above)
Comment (wedeling) [reply easyvvuq/analysis/sc_analysis.py]: I removed the commented subroutine

**Pull request merged**